### PR TITLE
Update Defibrillator presets

### PR DESCRIPTION
**Pull request by chnuessli**
In this PR, i want to change the presets in emergency=defibrillator. I think it would be better tagging on this type of node, when this presets will be showed. thank you

### CODE REVIEW

File: data/presets/emergency/defibrillator.json [hunk 1]
```diff
@@ -2,13 +2,14 @@
     "icon": "maki-defibrillator",
     "fields": [
         "indoor",
-        "ref",
```
Comment (peternewman) [inline data/presets/emergency/defibrillator.json:5]: Ref should be kept, it's used e.g. here:
https://osm.mathmos.net/defib/no-coords.html

Comment (chnuessli) [reply data/presets/emergency/defibrillator.json:5]: It still can be used, it's just not on the frond-page.
But either way, i can work this in again. Should i?

Comment (peternewman) [reply data/presets/emergency/defibrillator.json:5]: Personally I would have said so as it's used by at least one data consumer. Both post boxes and bus stops (probably all transport) seem to have some form of ref in their field list.

Comment (chnuessli) [reply data/presets/emergency/defibrillator.json:5]: Hi, i added the ref again to the editor and made some code corrections in the json format. this commit is added to the pr. https://github.com/chnuessli/id-tagging-schema/commit/b038c43c3a8976d364ccea1005be77dd8e1bc3fa

Comment (peternewman) [reply data/presets/emergency/defibrillator.json:5]: Thanks @chnuessli . FWIW, for me at least this PR seems to have lots of other random changed files too, I don't know if you just need to update or something.

**Comment by chnuessli** [comment]
Hi @peternewman yes, but i reverted the wrong commit... it was a problem with changes between windows/mac on the coding device 👎🏼 . But it was reverted, anyway the PR is untouched by the repo owner, i hope it will be merged soon.

**Comment by TurnrDev** [comment]
@chnuessli Can you rebase? It looks like your PR is deleting tests and touching the readme which is not ideal

**Comment by chnuessli** [comment]
@TurnrDev I will set up a new PR and close this one

**Comment by TurnrDev** [comment]
Moved to #135

**Pull request closed without merging**